apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/BufferOptions.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+
+namespace Google.Cloud.Diagnostics.Common
+{
+    /// <summary>
+    /// Types of buffers.
+    /// </summary>
+    internal enum BufferType
```

Comment:
Now that this is internal, all the public members using <see.../> references to the members should be removed/re-worded.